stripe/pubspec.lock

```diff
@@ -145,9 +152,9 @@ packages:
   stripe_ios:
     dependency: "direct main"
     description:
-      name: stripe_ios
-      url: "https://pub.dartlang.org"
-    source: hosted
+      path: "../stripe_ios"
```

Comment:
I think this will cause problems when we are publishing 

- [ ]  Check when running `pub publish --dry run`

Reply:
Yes, you are right, this will cause problems when we are publishing.
But currently it's not only the `stripe_ios` causing the problem, previous `pubspec.lock` that was checked in had dependencies on the `stripe_android` and `stripe_platform_interface` as well.
I notice that other Flutter plugins don't check-in their `pubspec.lock` on the Git, so I think I'll create an issue and do it in a separate PR if you don't mind @remonh87

Reply:
https://github.com/flutter-stripe/flutter_stripe/issues/51

Added this issue and will do it in a separate PR as we discussed offline ^^